packages/react/lib/render.js

```diff
@@ -7,10 +7,20 @@ const { Helmet } = require('react-helmet');
 
 function determineAssets(modules, stats) {
   const { entryFiles, vendorFiles, moduleFileMap } = stats;
-  const moduleFiles = modules.reduce(
-    (result, module) => [...result, ...moduleFileMap[module]],
-    []
-  );
+  const moduleFiles = modules.reduce((result, module) => {
+    if (!moduleFileMap[module]) {
+      if (process.env.NODE_ENV !== 'production') {
+        // eslint-disable-next-line no-console
+        console.warn(
```

Comment:
To be honest, I would prefer either silently swallowing this (as proposed by @toomuchdesign in #240) or throwing an error if we want to tackle this issue the way you propose.

If we wanted to introduce warnings for certain kinds of erroneous uses of `@untool`'s features, I would prefer us to introduce some sort of central mechanism (ahem, `hook`) instead of an ad-hoc implementation including `ENV` check.

Reply:
Fixing stats is probably the preferable option.
Thinking about it, if I'm not mistaken we can't reliably know if an affected module will eventually end up in an entry chunk or not (because it might be treeshaken later on in those cases where the imported module is only part of another file but not actually required)